### PR TITLE
Added tags pull step to drone config to show correct version hashes i…

### PR DESCRIPTION
**Pull request by kolaente**
As title.

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -69,6 +74,7 @@ pipeline:
     environment:
       TAGS: bindata sqlite sqlite_unlock_notify
     commands:
+      - git tag # To see if the tags were successfully pulled
```
Comment (zeripath) [inline .drone.yml]: Will this be here permanently? Or do you want to remove it after the first successful build premerging?

Comment (kolaente) [reply .drone.yml]: Just removed it, was just to verify the tags are pulled correctly.

Comment (techknowlogick) [reply .drone.yml]: Yes, please remove this before merge.

**Comment by kolaente** [comment]
Tags are apparently pulled correctly, so proper versions should work now. Though I'm not quite sure how to test this since it needs a release...

**Comment by zeripath** [comment]
Damn I don't think it works (from the first build log): 

```GO111MODULE=on go build -mod=vendor -i -v -tags 'bindata sqlite sqlite_unlock_notify' -ldflags '-s -w -X "main.MakeVersion=GNU Make 4.1" -X "main.Version=a96318d30" -X "main.Tags=bindata sqlite sqlite_unlock_notify"' -o gitea```

**Comment by kolaente** [comment]
From the build command, it appers to work now: https://drone.gitea.io/go-gitea/gitea/8634/1/10
```
GO111MODULE=on go build -mod=vendor -i -v -tags 'bindata sqlite sqlite_unlock_notify' -ldflags '-s -w -X "main.MakeVersion=GNU Make 4.1" -X "main.Version=1.9.0+dev-192-g6fbf87d2d" -X "main.Tags=bindata sqlite sqlite_unlock_notify"' -o gitea 
```

**Review by techknowlogick — APPROVED** [review]

**Comment by zeripath** [comment]
Please send backport to 1.8

**Comment by sapk** [comment]
Does this fix https://github.com/go-gitea/gitea/issues/6806 ?

**Comment by zeripath** [comment]
Well it depends on what they mean by a stable output

**Comment by zeripath** [comment]
But yes things should now have 1.9.0+dev prefixed and similarly for 1.8

**Comment by sapk** [comment]
It should also fix https://github.com/go-gitea/gitea/issues/4782 as on the tag the version is good event without the DRONE_TAG set :
```
/go/src/code.gitea.io/gitea # git checkout v1.8.0
HEAD is now at 799f5e05c 1.8.0 Changelog (#6678)
/go/src/code.gitea.io/gitea # git describe --tags --always | sed 's/-/+/' | sed 's/^v//'
1.8.0
```

**Comment by kolaente** [comment]
Backport is up:  #6840

**Pull request merged**